k6/README.md

```diff
@@ -0,0 +1,14 @@
+# K6 ReadMe
+
+## Introduction
+
+This readme is for how to install and us k6 for manual load testing
```

Comment:
typo: "use k6"

I think a good intro paragraph would explain what we are using k6 for, and why! So yes, generally performance testing, but also worth mentioning k6 is free and open-source 💚